src/hdusd/ui/render.py

```diff
@@ -96,6 +141,31 @@ def draw(self, context):
                  text=settings.data_source if settings.data_source else scene.name,
                  icon='NODETREE' if settings.data_source else 'SCENE_DATA')
 
+        if self.engine_type == 'FINAL' and settings.data_source and settings.data_source == 'NodeTree':
+
+            node_groups = bpy.data.node_groups
+            camera_source = ''
+
+            for ng in node_groups:
+                if ng.bl_idname != 'hdusd.USDTree':
+                    continue
+
+                stage = ng.get_output_node().cached_stage()
+                prims = tuple(stage.TraverseAll())
+
+                for prim in prims:
```

Comment:
This is not need, settings.camera_source has to be updated in usd_nodes/node_tree.py output_node_computed()